python/foglamp/plugins/south/systeminfo/systeminfo.py

```diff
@@ -218,13 +216,11 @@ def insert_reading(asset, time_stamp, data):
     try:
         time_stamp = utils.local_timestamp()
         get_system_info(time_stamp)
-    except asyncio.CancelledError :
-        pass
     except OSError as ex:
         _LOGGER.exception("Encountered System Error: {}".format(str(ex)))
```

Comment:
why there is no raise if OSError happens

Reply:
done